embedflag.go

```diff
@@ -1,18 +1,4 @@
 // +build embed
 
-package gorocksdb
-
-// #cgo CXXFLAGS: -std=c++11
-// #cgo CPPFLAGS: -I${SRCDIR}/../../cockroachdb/c-lz4/internal/lib
-// #cgo CPPFLAGS: -I${SRCDIR}/../../cockroachdb/c-rocksdb/internal/include
-// #cgo CPPFLAGS: -I${SRCDIR}/../../cockroachdb/c-snappy/internal
-// #cgo LDFLAGS: -lstdc++
-// #cgo darwin LDFLAGS: -Wl,-undefined -Wl,dynamic_lookup
-// #cgo !darwin LDFLAGS: -Wl,-unresolved-symbols=ignore-all -lrt
-import "C"
-
-import (
-	_ "github.com/cockroachdb/c-lz4"
-	_ "github.com/cockroachdb/c-rocksdb"
-	_ "github.com/cockroachdb/c-snappy"
-)
+// This instruction will trigger an explicit error during build time.
+"The embed build tag is no longer supported. Please refer to gorocksdb README file."
```

Comment:
This might, or might not be a good idea... What do you think about this information line?